dict_tools/__init__.py

```diff
@@ -7,6 +7,22 @@ def map_keys(function, dictionary):
 
 def map_values(function, dictionary):
     return {
-        key: function(value),
+        key: function(value)
+        for key, value in dictionary.items()
+    }
+
+
+def flattened_items(dictionary):
+    for key, value in dictionary.items():
+        if isinstance(value, (tuple, list)):
+            for subvalue in value:
+                yield (key, subvalue)
+        else:
+            yield (key, value)
```

Comment:
I was thinking for a while what bothered me about this and I think it's that I don't generally like when you have to have isinstance checks like this. I'd prefer this to _only_ operate on Dict[..., Iterable] types.

Reply:
You are very picky. No wonder you're single.

I made it do this so that you can do this:
```
mydict = {
	'no': 'pomegranates',
	'fbi': ('open', 'up'),
}
```
Instead of this:
```
mydict = {
	'no': ('pomegranates',),
	'fbi': ('open', 'up'),
}
```

Reply:
So, if I were to annotate the type of this in python 3, it would be, at best Dict[str, Union[str, Iterable[str]]], which is just a weird type to have. Generally, I'm against any structures that require an isinstance check, since that means your code is not well factored.

Reply:
i fixed it